src/sagemaker/model_monitor/clarify_model_monitoring.py

```diff
@@ -500,14 +524,15 @@ def suggest_baseline(
     # noinspection PyMethodOverriding
     def create_monitoring_schedule(
         self,
-        endpoint_input,
         ground_truth_input,
+        endpoint_input=None,
```

Comment:
changing the order of arguments might break for Cx's existing workflows.

Reply:
yea, but endpoint_input became optional, and non-none `ground_truth_input` has to come first.

Reply:
talked offline with @mufaddal-rohawala, we default `ground_truth_input` to `None` in the signature, however, we raise value error if it's None within the function.